### ISSUE TITLE
TextInput with multiline={true} does not support accessibility on iOS

### ISSUE THREAD

**Issue by laynemoseley** [facebook/react-native#13691]
### Description

A multiline `<TextInput>` does not support any sort of accessibility on iOS.

### Reproduction Steps and Sample Code

1. Add the following text input:
```
<TextInput multiline={true} />
```
2. Turn on VoiceOver
3. Navigate to the element
4. Observe that no text is spoken for the element.

### Solution

When multiline={false} is set, accessibility works as expected on iOS. `multiline={true}` should behave in the same way.

### Additional Information

* React Native version: latest
* Platform: iOS
* Development Operating System: MacOS
* Dev tools: Xcode, VSCode

**Comment by laynemoseley** [comment]
Here is a basic sample project that shows the issue.

[TextInputTest.zip](https://github.com/facebook/react-native/files/962862/TextInputTest.zip)

**Comment by prakashhiranandani** [comment]
I can also reproduce this. Any updates when this will be available :)

**Comment by prakashhiranandani** [comment]
We fixed above by adding below lines:
node_modules/react-native/Libraries/Text/RCTTextView.m
Add the below property to initWithEventDispatcher 
self.isAccessibilityElement = YES;
